lighthouse-core/audits/dobetterweb/doctype.js

```diff
@@ -6,6 +6,28 @@
 'use strict';
 
 const Audit = require('../audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on the doctype of a site. This descriptive title is shown to users when the site's doctype is set to HTML. */
+  title: 'Page has the HTML doctype',
+  /** Title of a Lighthouse audit that provides detail on the doctype of a site. This descriptive title is shown to users when the site's doctype is not set to HTML. */
+  failureTitle: 'Page is missing the HTML doctype',
```

Comment:
```suggestion
  failureTitle: 'Page lacks the HTML doctype, thus triggering quirks-mode',
```

joel gave feedback that it'd be helpful to more proactively communicate why the missing doctype is bad.  i agreed once i ran into this on my own.